### DISCUSSION TITLE
Why don't you start with the radio-browser.info DB?

### DISCUSSION THREAD

**Discussion by JensKorte** [jcorporation/webradiodb#412] [General]
From <https://www.radio-browser.info/>:Data license: public domain, software license: GPL, server software: open source

You should be allowed to reuse the DB from radio-browser. Writing a convert script could save you some work in adding the streams manually.

**Comment by goliv04053** [comment]
It's a good idea but I think he would need to filter all the radio who is already added on webradiodb.

**Comment by goliv04053** [comment]
But well, radio-browser.info DB would be a good beginning with filtering in webradioDB

**Comment by jcorporation** [comment]
A convert script would be possible, but automatically filtering out existing entries and the whole bunch of duplicate and incomplete entries is not possible. That’s why I decided to start with a clean base.

**Reply by JensKorte** [reply]
It is your DB and your decision.

**Reply by goliv04053** [reply]
What about the creation of a separated converted DB and and we migrate and separate the radios manually? And we also try to find ways of automating it.

**Reply by jcorporation** [reply]
That is possible, but who does the work?

**Comment by JensKorte** [comment]
Maybe someone is interested: <https://api.radio-browser.info/>
